### PR TITLE
Add Buoyant Cloud case-study

### PR DESCRIPTION
**Pull request by siggy**
Also add Buoyant as a Cortex adopter.

Signed-off-by: Andrew Seigner <andrew@sig.gy>

**What this PR does**:

Add a case study on Buoyant Cloud's Cortex integration. /cc @pracucci 

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/case-studies/buoyant-cloud.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## Linkerd x Prometheus
+
+In 2016, Linkerd 1.0 shipped with a [visualization module](https://linkerd.io/2016/10/04/a-service-mesh-for-kubernetes-part-i-top-line-service-metrics/#step-3-install-linkerd-viz) built with Prometheus and Grafana. With the release of [Linkerd 2.0 in 2018](https://linkerd.io/2018/09/18/announcing-linkerd-2-0/), Prometheus and Grafana were promoted to first-class components. That evolution was documented in a blog post entitled [Prometheus the Right Way](https://linkerd.io/2018/05/17/prometheus-the-right-way-lessons-learned-evolving-conduits-prometheus-integration/), and a KubeCon North America 2018 talk entitled [Linkerd 2.0, Now with Extra Prometheus](https://www.youtube.com/watch?v=bnDWApsH36Y), both co-authored by [Frederic Branczyk](https://twitter.com/fredbrancz) of Red Hat and [Andrew Seigner](https://twitter.com/siggy) of Buoyant.
```
Comment (gouthamve) [inline docs/case-studies/buoyant-cloud.md:14]: Frederic is not at redhat anymore. Maybe say Polar Signals (ex-redhat)? Not sure how to put it best.

**Review by gouthamve — APPROVED** [review]
I <3 <3 <3 this case-study!

Do you want to be added the list on our website as well? (requires editing `./website/data/adopters.yml` and putting the logo in `website/static/logos`)

**Comment by gouthamve** [comment]
image

How it looks now!

**Review by pracucci — APPROVED** [review]
Thanks! LveryGTM 👏 🙏

File: docs/case-studies/buoyant-cloud.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+This deep integration in Linkerd 2.0 provided out-of-the-box golden metrics, that is, success rate, request rate, and latency, across all your Kubernetes workloads.
+
+![Linkerd Grafana](/images/case-studies/bcloud-linkerd-grafana.png)
+_Linkerd 2.0 with integrated Prometheus and Grafana, out-of-the-box golden metrics_
+
+## Buoyant x Cortex
```
Comment (pstibrany) [inline docs/case-studies/buoyant-cloud.md:21]: I'm not sure what "Buoyant x Cortex" means. Is that multiplication?

**Review by pstibrany — APPROVED** [review]
❤️

File: docs/case-studies/buoyant-cloud.md (hunk 2 above)
Comment (siggy) [reply docs/case-studies/buoyant-cloud.md:21]: It is, I meant to imply "combining these two things is greater than the sum of the parts". =)

**Pull request merged**